style.css

```diff
@@ -1,14 +1,12 @@
 * {
   margin: 0;
   padding: 0;
+  text-decoration: none;
 }
 
```

Comment:
Fix the button hover to matches the one from the template.
![Screenshot from 2023-07-11 12-07-58](https://github.com/memicelis/my-portfolio/assets/118063058/cfda0a8b-581e-4a05-9538-a3b6d4487645)
![Screenshot from 2023-07-11 12-08-13](https://github.com/memicelis/my-portfolio/assets/118063058/9eb9df26-8553-4082-9059-22780ba62432)